README.md

```diff
@@ -79,7 +79,7 @@ Compiling:
 cd FCCDetectors
 mkdir build
 cd build
-cmake ../ -DCMAKE_INSTALL_PREFIX=../install -DCMAKE_BUILD_TYPE=RELWITHDEBINFO -DCMAKE_CXX_STANDARD=17
+cmake ../ -DCMAKE_INSTALL_PREFIX=../install -DWITH_DD4HEP=ON
```

Comment:
Sorry for the late comment but I don't think -DWITH_DD4HEP=ON is needed for these repositories

Reply:
Thanks for your comments @BrieucF. So for FCCAnalyses we use:
cmake ../ -DCMAKE_INSTALL_PREFIX=../install -DWITH_DD4HEP=ON
For FCCDetectors, k4SimGeant4 and k4RecCalorimeter we use:
cmake ../ -DCMAKE_INSTALL_PREFIX=../install 

Am I right?

Reply:
Yes!